tests/hooks/test_hive_hook.py

```diff
@@ -390,6 +390,11 @@ def test_table_exists(self):
             self.hook.table_exists(str(random.randint(1, 10000)))
         )
 
+    @mock.patch('random.shuffle')
+    def test_check_hms_clients_load_balance(self, mock_shuffle):
+        HiveMetastoreHook()._find_valid_server()
+        mock_shuffle.has_calls()
+
```

Comment:
```suggestion
    @mock.patch('airflow.hooks.hive_hooks.random.shuffle')
    @mock.patch('airflow.hooks.hive_hooks. HiveMetastoreHook.get_connections')
    def test_check_hms_clients_load_balance(self, mock_get_conn, mock_shuffle):
        HiveMetastoreHook()._find_valid_server()
        mock_shuffle.assert_called_once_with(mock_get_conn.return_value)

```
WDYT?

Reply:
LGTM. just one small change, mocking get_metastore_client as well to avoid error.

Reply:
@turbaszek Please take a look. Thank you!